Sources/PubNub/Subscription/SubscriptionSession.swift

```diff
@@ -316,6 +316,10 @@ public class SubscriptionSession {
               // No new request has been created so we'll reconnect here
               self?.reconnect(at: self?.previousTokenResponse?.timetoken)
             }
+          } else if let timetokenPayload = error.pubNubError?.affected.findFirst(by: PubNubError.AffectedValue.subscribe) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 123 characters](https://app.codacy.com/manual/PubNub/swift/pullRequest?prid=5635978)